docs/changelog-nl.rst

```diff
@@ -9,6 +9,211 @@ Changelog (NL)
+      de Generieke JSON- of Objecten-API-registratieplugins, en beschrijft de gegevensstructuur
+      geproduceerd door een van deze plugins.
+    - Alle gebruikers- en componentvariabelen zijn inbegrepen in het schema.
+    - De componentschemas bevatten validatieregels en een beschrijving indien beschikbaar.
+
+* [:backend:`5174`] De mogelijkheid toegevoegd om een omschrijving te configureren
+  voor 'zaakbetrokkenen' (registratoren, mede-ondertekenaars of partners) in de StUF-ZDS-plugin.
+* [:backend:`4877`] Ondersteuning toegevoegd voor het bijvoegen van een kopie van de
+  bevestigingse-mail(s) verstuurd naar de initiator in een aangemaakte zaak
+  in de ZGW API's and StUF-ZDS registraties.
+* [:backend:`5193`] `exp` claim toegevoeggd aan JWT in ZGW APIs.
+* [:backend:`5283`] De getoonde kolommen in de admin-formulierenlijst zijn opgeschoond
+  om de UX te verbeteren.
+
+**Bugfixes**
+
+* [:backend:`5394`] Een crash opgelost bij het opslaan van de DigiD- of eHerkenning-
+  configuratie in de admin
+* [:backend:`5041`] Probleem opgelost waarbij componenten met een punt in hun sleutel
+  niet toegevoegd werden aan de data van de Generieke JSON-registratie.
+* Probleem verholpen waarbij verborgen selectievakjes component onderdeel was van de
+  ingediende data als leeg object.
+* [:backend:`5326`] Fouten door onvoldoende geheugen tijdens de e-mailopschoning opgelost.
+* Het niet matchen van de standaardwaarde van de ``clearOnHide``-optie met de frontend opgelost.
+* [:backend:`5303`] Springende gebruikersvariabelen vanwege de auto-sort opgelost.
+* [:backend:`4401`] Oneindige omleiding door fout-geconfigureerde OIDC-authenticatiebackend opgelost.
+* [:backend:`5300`] Een regressie met geneste ingediende data in de vorige alpha release
+  is opgelost.
+* [:backend:`4933`] Ontbrekende Cosign v2-informatie toegevoegd voor registratie-e-mailsjablonen.
+* [:backend:`5245`] Een incorrecte variablekoppeling-configuratie wanneer er meerdere
+  registratiebackends beschikbaar zijn voor een form is opgelost.
+* [:backend:`5214`] Het niet gebruiken van de employee ID binnen de authenticatiecontext wanneer de organization-via-OIDC-plugin gebruikt wordt, is opgelost.
+* [:backend:`5238`] De volgorde van de formulierversies in de versiegeschiedenis is opgelost.
+* [:backend:`5263`] Dubbele encodering van data in de Generieke JSON-registratieplugin
+  is opgelost.
+* [:backend:`5202`] Afspraakinformatie onder het onderdeel inzendingen in de admin is verwijderd.
+* [:backend:`5207`] Twee bugs omtrent de referentielijsten-integratie zijn opgelost:
+
+    - Het genereren van JSON-schemas voor componenten die de referentielijsten als databron
+      gebruiken in de Generieke JSON-registratieplugin is opgelost.
+      gebruiken in de generieke JSON registratie plugin is opgelost.
+    - Het tonen van correcte items van niet-correcte tabellen voor componenten die referentielijsten
+      als databron gebruiken is opgelost.
```

Comment:
correct?

probably better to use actief/actieve?

Reply:
Changed to:
https://github.com/open-formulieren/open-forms/blob/a6ad8ec626981794613cc3248a373f074ea21404/docs/changelog-nl.rst?plain=1#L163